### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by Christelle-12**
In this milestone we :
-created  the popup with details about the project it appears when the user clicks (or taps) the see project  button
-With the X button the popup disappears.
- used javaScript array to store all of the information for all projects.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,449 +1,415 @@
...
-            </div>
+      <div id="popup-container" class="popup-container">
+            
         </div>
-    </div>
+        <section class="works" title="projects worked on" id="projects">
+            <section class="projects" title="project 1">
+                <img src="images/project-1.svg" class="project-1 project" alt="project-one" />
+                <img src="images/Snapshoot Portfolio.svg" class="project-1 project-1-desktop"
+                    alt="project-one" />
```
Comment (christianbiring1) [inline index.html:105]: - [ ] Kindly make sure all the elements on the work section are rendered dynamically via Javascript as you have already stored the project details in an array of objects.  In your markup file, we are supposed to have only the container of the work section and nothing else. Kindly correct that.

File: index.html [hunk 2]
```diff
@@ -1,449 +1,415 @@
...
+                        </ul>
+                    </section>
+                    <article class="intro-works">
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.
+                    </article>
+                    <ul class="languages">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
```
Comment (christianbiring1) [inline index.html:127]: - [ ] Kindly make sure all the elements on the work section are rendered dynamically via Javascript as you have already stored the project details in an array of objects.  In your markup file, we are supposed to have only the container of the work section and nothing else. Kindly correct that.

File: index.html [hunk 3]
```diff
@@ -1,449 +1,415 @@
...
-                </ul>
-                <div class="see-project">
-                    <a href="#"><button type="button" class="hex">See Project</button></a>
-                </div>
-            </div>
+                    <button class="see-project" title="see project" id="openpop-1">
+                        See Project
+                    </button>
+                </section>
+            </section>
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure all the elements on the work section are rendered dynamically via Javascript as you have already stored the project details in an array of objects.  In your markup file, we are supposed to have only the container of the work section and nothing else. Kindly correct that.

File: index.html [hunk 4]
```diff
@@ -1,449 +1,415 @@
...
+                    </section>
+                    <article class="intro-works intro-works-project1">
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.
+                    </article>
+                    <article class="intro-works intro-works-project2">
+                        Experimental content creation feature that allows users to add to
+                        an existing story over the course of a day without spamming their
+                        friends.
+                    </article>
```
Comment (christianbiring1) [inline index.html:231]: - [ ] Kindly make sure all the elements on the work section are rendered dynamically via Javascript as you have already stored the project details in an array of objects.  In your markup file, we are supposed to have only the container of the work section and nothing else. Kindly correct that.

File: index.html [hunk 5]
```diff
@@ -1,449 +1,415 @@
...
+                        <li class="lang-desk">Ruby on rails</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button class="see-project" title="see project" id="openpop-2">
+                        See Project
+                    </button>
+                </section>
+            </section>
+        </section>
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure all the elements on the work section are rendered dynamically via Javascript as you have already stored the project details in an array of objects.  In your markup file, we are supposed to have only the container of the work section and nothing else. Kindly correct that.

File: index.html [hunk 6]
```diff
@@ -1,449 +1,415 @@
...
+                        <h2 class="title title-desk">Facebook 360</h2>
+                        <ul class="categories categories-mobile" title="tech stack">
+                            <li class="item1">CANOPY</li>
+                            <li class="counter">
+                                <img class="img-counter" src="images/Counter.svg" alt="Counter" />Back End Dev
+                            </li>
+                            <li class="counter">
+                                <img class="img-counter" src="images/Counter.svg" alt="Counter" />2015
+                            </li>
+                        </ul>
```
Comment (christianbiring1) [inline index.html:318]: - [ ] Kindly make sure all the elements on the work section are rendered dynamically via Javascript as you have already stored the project details in an array of objects.  In your markup file, we are supposed to have only the container of the work section and nothing else. Kindly correct that.

File: index.html [hunk 7]
```diff
@@ -1,449 +1,415 @@
...
+                        Exploring the future of media in Facebook's first Virtual Reality
+                        app; a place to discover and enjoy 360 photos and videos on Gear
+                        VR.
+                    </article>
+                    <ul class="languages" title="programming languages">
+                        <li>html</li>
+                        <li class="lang-desk">Ruby on rails</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
```
Comment (christianbiring1) [inline index.html:345]: - [ ] Kindly make sure all the elements on the work section are rendered dynamically via Javascript as you have already stored the project details in an array of objects.  In your markup file, we are supposed to have only the container of the work section and nothing else. Kindly correct that.

File: index.html [hunk 8]
```diff
@@ -1,449 +1,415 @@
...
+                        <h2 class="title title-desk">Uber Navigation</h2>
+                        <ul class="categories categories-mobile" title="tech-stack">
+                            <li class="item1">CANOPY</li>
+                            <li class="counter">
+                                <img class="img-counter" src="images/Counter.svg" alt="Counter" />Back End Dev
+                            </li>
+                            <li class="counter">
+                                <img class="img-counter" src="images/Counter.svg" alt="Counter" />2015
+                            </li>
+                        </ul>
```
Comment (christianbiring1) [inline index.html:424]: - [ ] Kindly make sure all the elements on the work section are rendered dynamically via Javascript as you have already stored the project details in an array of objects.  In your markup file, we are supposed to have only the container of the work section and nothing else. Kindly correct that.

File: index.html [hunk 9]
```diff
@@ -1,449 +1,415 @@
...
+                        sign-ups required.
+                    </article>
+                    <article class="intro-works intro-works-project2">
+                        A smart assistant to make driving more safe, efficient, and fun by
+                        unlocking your most expensive computer: your car.
+                    </article>
+                    <ul class="languages" title="programming languages">
+                        <li>html</li>
+                        <li class="lang-desk">Ruby on rails</li>
+                        <li>css</li>
```
Comment (christianbiring1) [inline index.html:447]: - [ ] Kindly make sure all the elements on the work section are rendered dynamically via Javascript as you have already stored the project details in an array of objects.  In your markup file, we are supposed to have only the container of the work section and nothing else. Kindly correct that.

File: portfolio.js [hunk 10]
```diff
@@ -1,14 +1,198 @@
...
+
+data.forEach((item) => {
+  const { id } = item;
+
+  document.getElementById(`openpop-${id}`).addEventListener('click', () => {
+    document.getElementById(`popup-project-${id}`).classList.add('visible');
+  });
+
+  document.getElementById(`close-${id}`).addEventListener('click', () => {
+    document.getElementById(`popup-project-${id}`).classList.remove('visible');
```
Comment (christianbiring1) [inline portfolio.js]: - [ ] Kindly make sure When the user clicks (or taps) the button to check project details, the popup with details about that project appears i.e `photo`, `details`, `title`, `...` but as you may notice on all the project you only have one popup model content.

File: style.css [hunk 11]
```diff
@@ -1,575 +1,829 @@
...
 }
 
-.headline {
-  /* width: 100%; */
-  background-image: url("images/header-shapes.png");
-  background-size: 100% 550px;
-  border-radius: 0 0 0 100px;
-  display: flex;
-  flex-direction: column;
+.modal-mobile {
```
Comment (christianbiring1) [inline style.css:153]: - [ ] Also make sure you stick to the Figma design as much as possible. the spacing between elements(margin, padding) the typo(fonts) should be the same. As you may see your popup modal is not well-centered and the font doesn't match the Figma design.
image

File: style.css [hunk 12]
```diff
@@ -1,575 +1,829 @@
...
+  position: relative;
+  top: 0;
+  left: 0;
+  margin-top: 18px;
+  margin-left: 16px;
+  margin-right: 16px;
```
Comment (christianbiring1) [inline style.css:159]: - [ ] Also for a better user experience, please make sure on mobile, the user can see the all content on the popup modal.
`You may consider adding a scroll on the popup`

image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Christelle-12 @massazoum , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
- [ ] All linter checks suggested are setup correctly :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 

## Required Changes ♻️
- [ ]  `Kindly make sure all the elements in the work section are rendered dynamically via Javascript as you have already stored the project details in an array of objects. in that way adding projects in the future will be easy. In your markup file, we are supposed to have only the container of the work section and nothing else. Kindly correct that.`
Like so:
```html
<section class="project__container">
</section>
```
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Christelle-12, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  Clean and Organized code
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**